package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bbc-a11y",
-  "version": "2.2.6",
+  "version": "2.2.7",
```

Comment:
@andymsuk should this be included in the commit, or kept for the release?

Reply:
@EmmaJP - good spot. 
@sbeaumontweb Please can this be removed and we'll do it as part of the release process?

Reply:
Yep, sorry, complete muscle memory from working on morph modules for so long, haha.

Reply:
No worries! :)

Reply:
@sbeaumontweb  
_Bump_